W-16972662-self-managed-server-fa (#578)
mulesoft/docs-api-manager: modules/ROOT/pages/_partials/task-add-api-instance.adoc

... Select the *Version* for Mule runtime.
... Select the *Java version*.
.. Enter a *Proxy app name*.

> * *Hybrid:* Select this option if you want to use a Mule runtime running on an on-premises server that is registered in Runtime Manager.

Reviewer: ```suggestion
* *Hybrid:* Select this option if you want to use a Mule runtime instance running on an on-premises server that is registered in Runtime Manager.
```

+
See xref:runtime-manager::servers-create.adoc[Register a server in Runtime Manager] for more information.
+
.. Select a target from the list or click **Add server**.
.. Enter a *Proxy app name*.

> * *Self-managed Server:* Select this option if you want to create a JAR file to deploy an API proxy to a Mule runtime running on an on-premises server that is not registered in Runtime Manager. For more information, see xref:mule-runtime::mule-deployment-model.adoc[].

Reviewer: ```suggestion
* *Self-managed Server:* Select this option if you want to create a JAR file to deploy an API proxy to a Mule runtime instance running on an on-premises server that is not registered in Runtime Manager. For more information, see xref:mule-runtime::mule-deployment-model.adoc[].
```

====

> Deploying an API proxy to a self-managed server in API Manager does not deploy the proxy to your on-premise server. After you deploy the proxy in API Manager, deploy the proxy to your on-premise server:

Reviewer: ```suggestion
Deploying an API proxy to a self-managed server in API Manager doesn’t deploy the proxy to your on-premises server. After you deploy the proxy in API Manager, deploy the proxy to your on-premises server:
```

. xref:download-proxy-task.adoc[Download your new API proxy JAR file].

> . xref:mule-runtime::deploy-on-premises.adoc[Deploy the JAR file to your on-premise server].

Reviewer: ```suggestion
. xref:mule-runtime::deploy-on-premises.adoc[Deploy the JAR file to your on-premises server].
```
